Resources/Private/Templates/Backend/Report/NotificationStatusButtons.html

```diff
@@ -0,0 +1,19 @@
+{namespace nz=CuyZ\Notiz\ViewHelpers}
+
+<f:layout name="Default" />
+
+<f:section name="Content">
+    <nz:backend.module.link module="Administration" class="btn btn-default">
+        <core:icon identifier="actions-system-extension-configure" />
```

Comment:
You can now use the new ViewHelper `<nz:core.icon />`

Same thing on line 14.

Reply:
👍 

Needed the PR to be merged first. 😄